package.json

```diff
@@ -36,7 +36,7 @@
     "@types/method-override": "0.0.31",
     "@types/multer": "1.3.7",
     "@types/mysql": "0.0.34",
-    "@types/node": "^6.14.4",
+    "@types/node": "^10.14.4",
```

Comment:
Shouldn't this be `^8.x.x`, so that we don't get typings for functions that will eventually fail on the node 8 test worker?

Reply:
The idea is to target 10+ as it's what we use everywhere and is the active LTS and stops us from being limited to a major bump to make use of node 10 specific features if they become useful whilst at the same time keeping the test worker around so as not to accidentally break node 8 support without a good reason, eg you can break node 6 support by adding a trailing `,` in `function (a, b, c,) {}` and I'd like to avoid breaking for something like that

That said the "test" worker mainly just ensures the codebase can be built so it's of limited use for enforcing if something works or not and instead pinejs usually gets heavily tested via our api tests